servermanagement/startSignal.go

```diff
@@ -3,50 +3,51 @@ package servermanagement
 import (
 	"encoding/json"
 	"log"
-	"net/http"
-	"strings"
-	"sync"
 
+	"github.com/SUMUKHA-PK/Raft-Distributed-Consensus/servermanagement/routing"
 	"github.com/SUMUKHA-PK/Raft-Distributed-Consensus/types"
 )
 
 // StartSignal initiates the servers to behave
 // according to the raft protocol.
 func StartSignal(config types.Configuration, RaftServers map[string]types.RaftServer) error {
-	wg := &sync.WaitGroup{}
-	wg.Add(len(config.Servers))
-	designationMap := make(map[string]string)
+	designationMap := make(map[string]types.State)
 	for i := range config.Servers {
-		designationMap[config.Servers[i].IP+":"+config.Servers[i].Port] = "follower"
+		designationMap[config.Servers[i].IP+":"+config.Servers[i].Port] = types.State{
+			Name:        "follower",
```

Comment:
Name follower? It should rather be designation.

If we are storing names, I will add another field (But there were no names in the type.Server)

Reply:
Its in the `State` struct. State.Name is follower.

Reply:
Sure, we can remove it. It was just used for checks and tests till now.